### ISSUE TITLE
Xcode 15 beta - Stored properties cannot be marked unavailable with '@available' 

### ISSUE THREAD

**Issue by rizwan95** [ephread/Instructions#296]
## Describe the bug
When we try to install the pod in Xcode 15 beta, we are getting the error ```
Stored properties cannot be marked unavailable with @available```
 in the following properties

```
    // MARK: Renamed Public Properties
    @available(*, unavailable, renamed: "isStarted")
    public var started: Bool = false

    @available(*, unavailable, renamed: "isPaused")
    public var paused: Bool = false
```

## Screenshots
<img width="941" alt="image" src="https://github.com/ephread/Instructions/assets/10910174/6248cd3f-b465-4432-ad26-9b4b8a38fcef">

## Environment
 - Device: Any iOS device or simulator running iOS 17 beta. 
 - iOS version: iOS 17 beta
 - Dependency Manager: CocoaPods

**Comment by ephread** [comment]
Thanks for reporting the issue!

Tracked by #295

**Comment by scier** [comment]
Thanks for the quick fix! Are there plans for a new point release with this commit?

**Comment by kallipigous** [comment]
Forgive me but the swift package version of this doesn't seem to be fixed. Are there any plans tp update it?

**Comment by galijot** [comment]
@kallipigous this did get fixed, just a new release wasn't made. You can temporarily install my fork https://github.com/galijot/instructions until a new release is made.

**Comment by scier** [comment]
I realize this project is in maintenance mode, but wanted to check and see if there are plans to tag a new release before Xcode 15 is released next month? Or if it would be better to use a fork?

**Comment by bendigi** [comment]
> @kallipigous this did get fixed, just a new release wasn't made. You can temporarily install my fork https://github.com/galijot/instructions until a new release is made.

I'm using pods to install Instructions, how would I go about installing the fork? Thank in advance

**Comment by ephread** [comment]
I just released [2.3.0](https://github.com/ephread/Instructions/releases/tag/2.3.0) for the three package managers. Sorry for the delay, and thanks for the PRs!
